charts/sn-platform/templates/broker/_broker.tpl

```diff
@@ -36,12 +36,8 @@ Define the kop service url
 Define the web service url
 */}}
 {{- define "pulsar.web.service.url" -}}
-{{- if and .Values.tls.enabled .Values.tls.broker.enabled -}}
-https://{{ template "pulsar.broker.service" . }}.{{ template "pulsar.namespace" . }}.svc.cluster.local:{{ .Values.broker.ports.https }}
```

Comment:
why we want to remove this though, if user doesn't want to use Istio but still need mTls between components we should still support that right.